### PR TITLE
Add WAF Cloudflare custom error pages

### LINKED ISSUE #85: Add WAF custom error pages for Cloudflare
WAF Block + WAF Challenge for the moment.

Trigger: https://confluence.epfl.ch:8443/display/EXAPP/Cloudflare+-+Activation+et+configuration+du+WAF

### PR DESCRIPTION
**Pull request by obieler**
Fix #85 

**Note:** Possible to preview in Cloudflare (Cloudflare > Custom pages).

### CODE REVIEW

File: grunt/compile-handlebars.js [hunk 1]
```diff
@@ -20,6 +20,25 @@ module.exports = {
       YEAR: (new Date()).getFullYear().toString()
     }]
   },
+  cloudflareCustomPages: {
+    files: [{
+      cwd: 'src/cloudflare_custom_pages',
```
Comment (williambelle) [inline grunt/compile-handlebars.js]: I prefer to use `errors` folder

File: src/cloudflare_custom_pages/cf-waf-block.html [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+              </svg>
+            </a>
+          </li>
+          <li class="breadcrumb-item active" aria-current="page">Blocked</li>
+        </ol>
+      </nav>
+    </div>
+
+    <div class="main-container">
+      <!-- Main... -->
```
Comment (williambelle) [inline src/cloudflare_custom_pages/cf-waf-block.html]: I think we can remove this comment.

File: src/cloudflare_custom_pages/cf-waf-challenge.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+              </svg>
+            </a>
+          </li>
+          <li class="breadcrumb-item active" aria-current="page">Challenge CAPTCHA required</li>
+        </ol>
+      </nav>
+    </div>
+
+    <div class="main-container">
+      <!-- Main... -->
```
Comment (williambelle) [inline src/cloudflare_custom_pages/cf-waf-challenge.html]: I think we can remove this comment.

**Comment by williambelle** [comment]
In the js console:

```
cf-waf-block.html:214 Uncaught TypeError: $(...).selectize is not a function
    at m (cf-waf-block.html:214)
    at HTMLDocument.<anonymous> (cf-waf-block.html:214)
    at e (cf-waf-block.html:150)
    at t (cf-waf-block.html:150)
```

**Comment by williambelle** [comment]
Do we need favicons and meta viewport ?

File: grunt/compile-handlebars.js (hunk 1 above)
Comment (obieler) [reply grunt/compile-handlebars.js]: Yes, better

File: src/cloudflare_custom_pages/cf-waf-block.html (hunk 2 above)
Comment (obieler) [reply src/cloudflare_custom_pages/cf-waf-block.html]: yes

File: src/cloudflare_custom_pages/cf-waf-challenge.html (hunk 3 above)
Comment (obieler) [reply src/cloudflare_custom_pages/cf-waf-challenge.html]: yes

**Comment by obieler** [comment]
> Do we need favicons and meta viewport ?

yes, added.

**Comment by obieler** [comment]
Fonctionne **que** avec la modification suivante dans elements: https://github.com/epfl-idevelop/elements/pull/401.

**Pull request merged**